.github/workflows/lido-ci.yml

```diff
@@ -0,0 +1,46 @@
+---
+  name: "Test Proofs"
+  on:
+    workflow_dispatch:
```

Comment:
Lets remove workflow_dispatch for now. 

And if we can request from them how often they wish to run the tests, if it's on Every PR change or just on merge to a specific branch. This will help best manage compute usage.